src/plugins/saved_objects_management/public/lib/bulk_delete_objects.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { HttpStart } from '@kbn/core/public';
+import { SavedObjectError } from '@kbn/core-saved-objects-common';
+
+interface SavedObjectDeleteRequest {
+  id: string;
+  type: string;
+}
```

Comment:
NIT/optional: avoid having 675 similar tuple types in our codebase, use `SavedObjectTypeIdTuple` instead

https://github.com/elastic/kibana/blob/0645a3ba388f81fc6b43136e64ad53614101e947/packages/core/saved-objects/core-saved-objects-common/src/types.ts#L14-L19